cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/compute/GcpReservedIpResourceBuilder.java

```diff
@@ -81,8 +105,7 @@ public CloudResource delete(GcpContext context, AuthenticatedContext auth, Cloud
             Operation operation = compute.addresses().delete(projectId, region, resource.getName()).execute();
             return createOperationAwareCloudResource(resource, operation);
         } catch (GoogleJsonResponseException e) {
-            exceptionHandler(e, resource.getName(), resourceType());
-            return null;
+            throw exceptionHandlerWithThrow(e, resource.getName(), ResourceType.GCP_RESERVED_IP);
```

Comment:
Here, again, we're causing a thrown exception when the resource isn't found, whereas before we weren't throwing an exception in that case. Why the change?

Reply:
I had made this change with the other static methods, no longer needed with injection.